src/types/inventory/productKnowledge/productKnowledge.ts

```diff
@@ -70,7 +70,7 @@ export interface ProductDefinition {
 export interface ProductKnowledgeBase {
   id: string;
   slug: string;
-  alternate_identifier?: string;
+  alternate_identifier: string | null;
```

Comment:
this is incorrect as per backend! the previous one was right. it is optional in backend.



<img width="728" height="445" alt="image" src="https://github.com/user-attachments/assets/2b7e0463-8cd7-4a42-8a71-90f72fe461c8" />